src/bot/services/task.py

```diff
@@ -0,0 +1,22 @@
+import contextlib
+from typing import Generator, List
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import joinedload
+
+from src.core.db.db import get_session
+from src.core.db.models import Task
+
+
+class TaskService:
+    """Сервис бота для работы с моделью Task."""
+
+    def __init__(self, sessionmaker: Generator[AsyncSession, None, None] = get_session):
+        self._sessionmaker = contextlib.asynccontextmanager(sessionmaker)
+
+    async def get_user_tasks(self) -> List[Task]:
```

Comment:
На версии 3.11 это выглядит иначе 
```suggestion
    async def get_user_tasks(self) -> list[Task]:
```

Reply:
Поправил, но тут не от версии питона же List?
Просто в учебных использовали List ане list в аннотации.
Хотя from typing import List есть в прошлом боте =)
> На версии 3.11 это выглядит иначе